src/dal/Jalasoft.TeamUp.Projects.DAL/ProjectsMongoDbRepository.cs

```diff
@@ -39,5 +39,12 @@ public Project GetById(Guid id)
 
             return project;
         }
+
+        public Project DeleteById(Guid id)
+        {
+            Project project = collection.FindOneAndDelete(x => x.Id == id);
+
+            return project;
```

Comment:
do we need to return a project in this case? since we a re not returning any Project to the UI could this be a void?

Reply:
done